pkg/grillelasticsearch/stubs.go

```diff
@@ -49,3 +49,24 @@ func (ge *ElasticSearch) UpsertItem(index, docId, data string) grill.Stub {
 		return nil
 	})
 }
+
+func (ge *ElasticSearch) AddScript(name string, script string) grill.Stub {
+	return grill.StubFunc(func() error {
+		req := esapi.PutScriptRequest{
+			Body:       strings.NewReader(script),
+			DocumentID: name,
+		}
+		res, err := req.Do(context.Background(), ge.elasticSearch.Client)
+		defer res.Body.Close()
```

Comment:
move this after the error check. in case of error, res can be nil

Reply:
ok